### PR TITLE
feat(tests): Add test vectors generated via python script.

### PR DESCRIPTION
**Pull request by rfk**
This adds a python script to generate some test vectors given specific values at each step of the login flow.  I've used it to add a couple more tests, and to provide concrete values in the proposal document at https://docs.google.com/document/d/1IvQJFEBFz0PnL4uVlIvt8fBS_IPwSK-avK0BRIHucxQ/edit#heading=h.d8itxwk4ovta

WIP while I see if the tests actually pass :-)

### CODE REVIEW

**Comment by rfk** [comment]
Yessss...test passing, @vladikoff r?

File: test/deriver/ScopedKeys.js [hunk 1]
```diff
@@ -38,6 +38,20 @@ describe('ScopedKeys', function () {
       });
   });
 
+  it('should match the output of test vectors generated via python script', () => {
```
Comment (vladikoff) [inline test/deriver/ScopedKeys.js]: 👍

File: test/generate_test_vectors.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+#
+# This script is designed as an independent implementation of the FxA
+# scoped keys protocol, instantiated with one specific deterministic
+# set of values for all out-of-band or randomly-generated data.  It
+# runs the protocol and prints out the exected values of various
+# intermediate states along with the final JWE encrypted key bundle.
+#
+# To run it, you will need to:
+#
+#    > pip install jwcrypto hkdf
```
Comment (vladikoff) [inline test/generate_test_vectors.py]: 👍

**Comment by vladikoff** [comment]
Awesome!

File: test/generate_test_vectors.py [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+        "kid": "{}-{}".format(key_rotation_timestamp, b64url_encode(kSfp)),
+        "k": b64url_encode(kS),
+        "kty": "oct"
+    }
+})
+
+print "keys_bundle:\t{}".format(keys_bundle)
+
+# Encrypt it into a JWE using an ephemeral key.
+# We do some slight hackery here to make the python `jwcrypto` module
```
Comment (vladikoff) [inline test/generate_test_vectors.py]: oh well!

**Pull request merged**